### PR TITLE
chore(gen): docs content review 2024-08-26

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: dedibox/hardware/how-to/configure-hardware-raid-console.mdx [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ---
 meta:
-  title: How to configure your hardware RAID from the console
-  description: This page explains how to configure your hardware RAID from the console
+  title: How to configure your Scaleway Dedibox hardware RAID from the console
+  description: This page explains how to configure your Scaleway Dedibox hardware RAID from the console
 content:
-  h1: How to configure your hardware RAID from the console
-  paragraph: This page explains how to configure your hardware RAID from the console
+  h1: How to configure your Scalewat Dedibox hardware RAID from the console
```
Comment (SamyOubouaziz) [inline dedibox/hardware/how-to/configure-hardware-raid-console.mdx:6]:
```suggestion
  h1: How to configure your Scaleway Dedibox hardware RAID from the console
```

**Review by SamyOubouaziz — APPROVED** [review]

File: compute/instances/how-to/use-boot-modes.mdx [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
   - compute
 ---
 
-You can boot your Instances using a local boot, rescue image, or `cloud-init`. This page explains how to use each of these boot modes.
+You can boot your Instances using local boot or a resuce image. This page explains how to use each of these boot modes.
 
 <Message type="tip">
-  You can choose to use **local boot** or **cloud-init** when [creating an Instance](/compute/instances/how-to/create-an-instance/) via **Advanced settings**. This enables you to control the mode of the Instance's very first boot.
+  - You can choose to use **cloud-init** when [creating an Instance](/compute/instances/how-to/create-an-instance/) via **Advanced settings**. This enables you to control the mode of the Instance's very first boot.
+  - Your Instance must be [powered off](/compute/instances/how-to/power-off-instance/) in order to change the boot mode of it.
```
Comment (RoRoJ) [inline compute/instances/how-to/use-boot-modes.mdx:20]:
```suggestion
  - Your Instance must be [powered off](/compute/instances/how-to/power-off-instance/) in order to change its boot mode.
```

File: console/account/how-to/resolve-abuse.mdx [hunk 3]
```diff
@@ -1,13 +1,13 @@
 ---
 meta:
-  title: How to resolve abuse tickets and cases
+  title: How to resolve abuse tickets and cases in Scaleway
```
Comment (RoRoJ) [inline console/account/how-to/resolve-abuse.mdx:3]:
```suggestion
  title: How to resolve Scaleway abuse tickets and cases
```

File: console/account/how-to/resolve-abuse.mdx [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
   description: Take actions to resolve abuse issues in Scaleway.
 content:
-  h1: How to resolve abuse tickets and cases
+  h1: How to resolve abuse tickets and cases in Scaleway
```
Comment (RoRoJ) [inline console/account/how-to/resolve-abuse.mdx:6]:
```suggestion
  h1: How to resolve Scaleway abuse tickets and cases
```

File: dedibox-network/rpn/reference-content/private-networks-overview.mdx [hunk 5]
```diff
@@ -7,79 +7,62 @@ content:
...
 
-The internet was initially designed to enable a few academics and researchers to share data between themselves. As such, security was not a paramount concern. Of course, over time the internet grew, with more and more private companies and personal users taking it up, until today it has become the omnipresent global network we all know, linking billions of machines and people worldwide. While technologies have evolved and data-protection mechanisms to secure our connections have become more sophisticated, we should never forget that the internet remains essentially a public network. This means that infrastructure attacks like Distributed Denial-of-Service or Brute force can always stand to pose a threat to your infrastructure and your data.
-​
+The Internet was initially designed for academic data sharing, with minimal focus on security. As it evolved into the global network we rely on today, it brought about new challenges, including exposure to Distributed Denial-of-Service (DDoS) attacks and other threats. Despite advancements in security technologies, the Internet remains a public network where risks persist.
```
Comment (RoRoJ) [inline dedibox-network/rpn/reference-content/private-networks-overview.mdx:19]:
```suggestion
The Internet was initially designed for academic data sharing, with minimal focus on security. As it evolved into the global network we rely on today, new challenges arose, including exposure to Distributed Denial-of-Service (DDoS) attacks and other threats. Despite advancements in security technologies, the Internet remains a public network where risks persist.
```

File: dedibox-network/rpn/reference-content/private-networks-overview.mdx [hunk 6]
```diff
@@ -7,79 +7,62 @@ content:
...
 
 ### Building Private Networks on Scaleway
 
-The **Private Networks** feature is available for all Instance types in the Scaleway Ecosystem. It allows Instances within an Availability Zone to communicate safely in an isolated and secured network, inaccessible from the internet. Private Networks are a LAN-like layer 2 Ethernet network.
-​
+Scaleway offers the **Private Networks** feature for all Instances, enabling secure, isolated communication within an Availability Zone, independent of the Internet. These Private Networks function as layer 2 Ethernet networks.
 
-Each Instance assigned to a Private Network instantly gets a virtual network interface connected to the Private Network (without needing a reboot). You can think of this as a virtual version of the second network card installed in the Dedibox servers. This interface allows the Instance to reach other Instances in the same Private Network.
-​
+Instances connected to a Private Network automatically gain a virtual network interface, facilitating seamless internal communication without requiring a reboot. Multiple private networks can be assigned to each Instance, allowing for flexible and complex network configurations.
```
Comment (RoRoJ) [inline dedibox-network/rpn/reference-content/private-networks-overview.mdx:54]:
```suggestion
Instances connected to a Private Network automatically gain a virtual network interface, facilitating seamless internal communication without requiring a reboot. Multiple Private Networks can be assigned to each Instance, allowing for flexible and complex network configurations.
```

File: dedibox/kvm-over-ip/how-to/supermicro.mdx [hunk 7]
```diff
@@ -1,10 +1,10 @@
...
   title: How to use the Supermicro KVM
-  description: This page explains how to use the KVM device of SuperMicro servers
+  description: This page explains how to use the KVM device of SuperMicro servers on Scaleway Dedibox.
 content:
   h1: How to use the Supermicro KVM
-  paragraph: This page explains how to use the KVM device of SuperMicro servers
+  paragraph: This page explains how to use the KVM device of SuperMicro servers on Scaleway Dedibox.
 tags: kvm supermicro
 dates:
   validation: 2024-02-19
```
Comment (RoRoJ) [inline dedibox/kvm-over-ip/how-to/supermicro.mdx:10]:
```suggestion
  validation: 2024-08-27
```

File: dedibox/hardware/how-to/configure-hardware-raid-console.mdx [hunk 8]
```diff
@@ -1,10 +1,10 @@
...
+  title: How to configure your Scaleway Dedibox hardware RAID from the console
+  description: This page explains how to configure your Scaleway Dedibox hardware RAID from the console
 content:
-  h1: How to configure your hardware RAID from the console
-  paragraph: This page explains how to configure your hardware RAID from the console
+  h1: How to configure your Scaleway Dedibox hardware RAID from the console
+  paragraph: This page explains how to configure your Scaleway Dedibox hardware RAID from the console
 tags: dedibox raid 
 dates:
   validation: 2024-02-12
```
Comment (RoRoJ) [inline dedibox/hardware/how-to/configure-hardware-raid-console.mdx:10]:
```suggestion
  validation: 2024-08-27
```

File: dedibox/dedicated-servers/concepts.mdx [hunk 9]
```diff
@@ -1,26 +1,26 @@
...
+  title: Dedibox dedicated servers - Concepts
+  description: This page explains all the concepts related to Scaleway Dedibox dedicated servers.
 content:
-  h1: Dedicated servers - Concepts
-  paragraph: This page explains all the concepts related to Dedibox
+  h1: Dedibox dedicated servers - Concepts
+  paragraph: This page explains all the concepts related to Scaleway Dedibox dedicated servers.
 tags: dedibox acl dedibackup os-type ipmi partitioning suchard vsphere esxi hostname
 dates:
   validation: 2024-02-19
```
Comment (RoRoJ) [inline dedibox/dedicated-servers/concepts.mdx:10]:
```suggestion
  validation: 2024-08-27

File: dedibox-network/rpn/concepts.mdx [hunk 10]
```diff
@@ -1,7 +1,7 @@
 ---
 meta:
-  title: RPN - Concepts
-  description: This page explains all the concepts related to RPN
+  title: Scaleway Dedibox RPN - Concepts
+  description: This page explains all the concepts related to RPN on Scaleway
```
Comment (ldecarvalho-doc) [inline dedibox-network/rpn/concepts.mdx:4]: is it normal you didnt change the date?

File: dedibox/hardware/how-to/configure-dell-perc-h200.mdx [hunk 11]
```diff
@@ -1,10 +1,10 @@
 ---
 meta:
   title: How to configure the DELL PERC H200 RAID controller
```
Comment (ldecarvalho-doc) [inline dedibox/hardware/how-to/configure-dell-perc-h200.mdx:3]: date not updated here either

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**